src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -290,7 +290,7 @@ protected function loadFile($file)
         try {
             $configuration = $this->yamlParser->parse(file_get_contents($file));
         } catch (ParseException $e) {
-            throw new \InvalidArgumentException(sprintf('The file "%s" does not contain valid YAML.', $file), 0, $e);
+            throw new InvalidArgumentException(sprintf('The service file "%s" is not valid. %s', $file, $e->getMessage()), 0, $e);
```

Comment:
That's what I used to have but a `.` is more consistent with other messages thrown from this class.